javascript/main.js

```diff
@@ -1,3 +1,7 @@
+//= require jquery
+//= require jquery_ujs
+//= require popper
+
```

Comment:
@FionaDL shouldn't this file be moved to `vendor/assets` as well? I don't think you will be able to use it from `dash` if it's here.